packages/web3-eth-miner/types/index.d.ts

```diff
@@ -17,13 +17,12 @@
  * @date 2019
  */
 
-import {Accounts} from 'web3-eth-accounts';
 import {provider} from 'web3-providers';
-import {AbstractWeb3Module, Web3ModuleOptions, Hex} from 'web3-core';
+import {AbstractWeb3Module, Web3ModuleOptions} from 'web3-core';
 import * as net from 'net';
 
 export class Miner extends AbstractWeb3Module {
-    constructor(provider: provider, net?: net.Socket|null, options?: Web3ModuleOptions, accounts?: Accounts|null);
+    constructor(provider: provider, net?: net.Socket|null, options?: Web3ModuleOptions, accounts?: any);
 
```

Comment:
this should be ``Accounts|null`` and not any.

Reply:
Ya, you are correct @nivida. But we had a discussion about it. @joshstevens19 suggested that if it's only used in typing then we shouldn't make accounts as a dependency of this module. That's why I changed it to any.

Reply:
@nivida Please check this [comment](https://github.com/ethereum/web3.js/pull/2660#discussion_r275371902).

Reply:
Thinking about this more we may as well move `Accounts` interface itself `web3-core` so it can be shared without adding any dependencies or not supplying solid typing's... thoughts?

Reply:
Yes, I was also thinking about the same @joshstevens19.

Reply:
I think provider interface should also be added to **web-core** typings to reduce dependency.

Reply:
Have you only brought provider in for typings? If so yes I agree. We should never bring a dependency in just for a type, if this is the case we have a common place for it in web3-core 👍

Reply:
No, I think we have to put web3-providers. It got used in multiple places. But we can export `Account` type into `web3-core`.